documentation/modules/exploit/multi/http/jenkins_script_console.md

```diff
@@ -0,0 +1,66 @@
+## Vulnerable Application
+
+  Jenkins can be downloaded from [jenkins.io](https://jenkins.io/) where
+  binaries are available for a variety of operating systems. Both LTS and weekly
+  builds are available.
+
+  Default settings have the script console enabled and require a valid user
+  account on order to access it. A known account can be used with this module by
+  setting the `USERNAME` and `PASSWORD` options.
+
+## Verification Steps
+
+  Example steps in this format:
+
+  1. Install the application
+  1. Start msfconsole
+  1. Do: ```use exploit/multi/http/jenkins_script_console```
+  1. Do: ```set RHOST [target host]```
+  1. Do: ```set TARGET [target id]```
+  1. Do: ```exploit```
+  1. You should get a shell.
+
+## Options
+
+  **PASSWORD**
+
+  A password to an account that has access to the script console. This is only
+  necessary if the Jenkins instance has been configured to require
+  authentication.
+
+  **TARGETURI**
+
+  The path to the target instance of Jenkins.
+
+  **USERNAME**
+
+  A username to an account that has access to the script console. This is only
+  necessary if the Jenkins instance has been configured to require
+  authentication.
+
+## Scenarios
+
+  Example usage against a Linux target server.
```

Comment:
I like to give more details here, `an Ubuntu xx.xx x64 server running Jenkins xxxx`